fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -364,6 +364,13 @@ public Cost visitPhysicalHashJoin(
             } else if (probeStats.getWidthInJoinCluster() < buildStats.getWidthInJoinCluster()) {
                 leftRowCount += 1;
             }
+            if (probeStats.getWidthInJoinCluster() == buildStats.getWidthInJoinCluster()
+                    && probeStats.computeSize() < buildStats.computeSize()) {
+                // When the number of rows and the width on both sides of the join are the same,
```

Comment:
The case has the same row count but different width, so we should use the "rowCount" and "computeTupleSize" to judge, instead of computeSize?

Reply:
There is an equal row count condition outside already. And the computeTupleSize is private. When row count is same, the computSize is same to computeTupleSize

Reply:
LGTM